backend/routes/leaderboard.ts

```diff
@@ -0,0 +1,60 @@
+import { Router } from 'express';
+import { AppDataSource } from '../AppDataSource';
+import { Leaderboard } from '../entities/Leaderboard';
+
+
+const baseUrl = '/leaderboard';
+const repository = AppDataSource.getRepository(Leaderboard);
+
+//route
+export const leaderBoardRoute = Router();
+
+
+leaderBoardRoute.get('/leaderboard/:id', (req, res) => {
+    const userId = Number.parseInt(req.params.id);
+
+    repository
+        .findOne({
+            where: { id: userId },
+        })
+        .then((leaderboard) => {
+            res.status(200).json(leaderboard);
+        })
+        .catch((error) =>
+            res.status(404).json({ errorMessage: `${userId} doesn't exist`, error: error }),
+        );
+});
+
```

Comment:
Do we really need both of these to get single user?

Reply:
Yes we might need them later